doc/crm.8.adoc

```diff
@@ -1551,7 +1550,8 @@ Example:
 info apache
 info ocf:pacemaker:Dummy
 info stonith:ipmilan
-info pengine
+info cluster
```

Comment:
```suggestion
```

If you agree, we could delete and replace "crm ra info cluster" to "crm configure property info".

Reply:
I prefer not to hide it from help info, just don't show it from completion result